scripts/zones/Port_San_dOria/npcs/Phersula.lua

```diff
@@ -8,7 +8,6 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-    player:startEvent(525063)
```

Comment:
lol what even was this

Reply:
Talks about Mog Houses, surprisingly?

![image](https://user-images.githubusercontent.com/8849608/205248100-1bee412a-0502-40fa-92b5-e9bc64953d6b.png)